packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -110,10 +109,13 @@ class _EditProductPageState extends State<EditProductPage> {
                       )
                     ],
                   ),
-                _ListTitleItem(
-                  title: appLocalizations.edit_product_form_item_details_title,
-                  subtitle:
-                      appLocalizations.edit_product_form_item_details_subtitle,
+                SmoothListTileCard.icon(
```

Comment:
> Because `_ListTileItem` would have been just a wrapper for `SmoothListTileCard`

That's exact, that's the purpose of methods. And just in case we want a bolder text on this page, we can do that in just one place, in `_ListTileItem`. I'm not saying that your intentions are bad, but I'm old fashioned: I do read code. I read _every_ change, with my limited brain bandwidth. In this PR you changed 18 files. If you needed to change 18 files, fair enough. But as a reviewer if I can avoid to read twice as many lines just because the developer coded the same thing but just differently with a tiny added value, that's better.
How do I do when confronted with a similar case as a developer: I push two distinct PRs. The first one is smart and the added value is there. The reviewers review, back and forth, the code is edited and merged, good. Then I push a second PR. The stupid one, where I just added a comma or moved files. For this PR I don't even ask for review and merge it immediately.

> I don't really know a lot about flutter performance but that would seem a waste.

You're marginally right: there's probably a minimal impact on performances. But I don't think we heard complains about performances on that screen in particular. If you started that screen from scratch, of course why not implementing that way.

> If you prefer I can change it back to `_ListTileItem` without problem

Please do. This way we'll focus on what is specifically interesting in your PR.